packages/cacti-plugin-copm-fabric/README.md

```diff
@@ -0,0 +1,55 @@
+# @hyperledger/cacti-plugin-copm-fabric
+
+This cactus plugin implements a connectRPC server for the fabric COPM implementation.
+
+Command documentation as OpenAPI:
+https://jenniferlianne.github.io/cacti/references/openapi/cacti-copm-core_openapi/
+
+These endpoints require the following:
+
+- weaver relays and drivers to be deployed on the network
+- chaincode contracts for 'asset exchange' and 'asset transfer' to be deployed on the fabric network
+  
+Please see https://hyperledger.github.io/cacti/weaver/introduction/.
+
+
+# Usage
+
+## Installation
+
+Yarn: 
+
+    yarn add --exact @hyperledger/cacti-plugin-copm-fabric
+
+
+## Configuration
+
+The following application-specific interfaces must be implemented:
+
+-  FabricConfiguration
+   -    getConnectionProfile(orgKey: string): object;
+   -    getContractContext(orgKey: string): Promise<FabricContractContext>;
+   -    getOrgWallet(orgKey: string): Promise<Wallet>;
+
+-  InteropConfiguration (from cacti-copm-common)
+   -    getLocalRelayConfig(orgKey: string): LocalRelayConfig;
+   -    getRemoteNetworkConfig(remoteOrgKey: string): RemoteNetworkConfig;
+
+  These implementations are then supplied to the plugin constructor. 
+
+## Development
+
+A Makefile is provided which will build a docker weaver network with the following commands:
+
+- make setup
+  - build all weaver components
+- make pledge-network
+  - makes a network for running pledge/claim (asset transfer)
+- make lock-network
+  - makes a network for running lock/claim (asset exchange)
+- make clean-network
+  - tear down the current network
+  
+The asset exchanges and asset transfer network modes are currently mutually exclusive.
```

Comment:
I see the `pledge-network` and `lock-network` targets in the `Makefile` doing the same thing currently. I guess you will differentiate them in a later PR?

Also, I'd like to understand why a network can be instantiated only with on capability (pledge/lock) and not the other. In the legacy Weaver test networks, we launch different example applications (e.g., the `simpleasset` and `simpleassettransfer` chaincodes) exclusively to demonstrate the different protocols (asset transfer, asset exchange). But there's no reason why we can't have both running on a network at the same time (a Fabric network can have any number of chaincodes deployed.) The common Fabric chaincode (from which you are drawing the COPM functions) contains the complete set of capabilities, which can be invoked on demand (just like we envision for the COPM).

Anyway, this is not a blocker for the current PR. We can discuss offline.

Reply:
The lock-network and pledge-network targets differ in the chaincode variable.  While it's true that fabric can support multiple contracts, the weaver makefile targets are set up to deploy a single application contract.  I looked at changing them, but one of the things the makefiles do is set up fabric-cli configuration with the specified single contract, so changes would need to be made in several places.

Reply:
The fabric-cli was originally meant to be a stopgap tool for testing functionality, but then acquired a life of its own. Still, it's not part of the reusable Weaver functionality (which is basically protos + core + sdk), so we shouldn't let the former's limitations seep into the latter.

Let's set this issue aside for now, but we do need to allow any and all COPM functions to be deployed in a given Fabric network. Can you open an issue to track that? We can list what needs to be done in that thread.